tests/drop_packets/test_drop_counters.py

```diff
@@ -238,6 +238,7 @@ def set_mtu(cls, mtu, iface):
             else:
                 raise Exception("Unsupported interface parameter - {}".format(iface))
             cls.iface = iface
+            time.sleep(3)
```

Comment:
Is it possible to use the `wait_until` method from `common/utilities` to poll `ifconfig` to see if the MTU has been updated yet or not?

Reply:
Great suggestion. Updated. Thanks.